src/mrinufft/operators/base.py

```diff
@@ -332,20 +332,23 @@ def compute_density(self, method=None):
 
         Parameters
         ----------
-        method: str or callable or array or dict
+        method: str or callable or array or dict or bool
             The method to use to compute the density compensation.
             If a string, the method should be registered in the density registry.
             If a callable, it should take the samples and the shape as input.
             If a dict, it should have a key 'name', to determine which method to use.
             other items will be used as kwargs.
             If an array, it should be of shape (Nsamples,) and will be used as is.
+            If a bool, it will enable or disable the density compensation.
         """
         if isinstance(method, np.ndarray):
             self.density = method
             return None
         if not method:
             self.density = None
             return None
+        if method is True:
```

Comment:
```suggestion
        if method is True and self.backend is in ['tfnufft', 'gpunufft', 'torchkbnufft']:
```

As we dont support `pipe` for other classes.